CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 4.8.0
+
+* Added `NotificationsAPIClient.send_precompiled_letter_notification()`
+    * The client can now send PDF files which conform to the Notify printing template
+    Send a File object or a base64 encoded string
```

Comment:
this line needs a bulletpoint too, the markdown screws up

Reply:
OK